private-ai-services/paisconfiguration.yaml

```diff
@@ -0,0 +1,57 @@
+# Example PAISConfiguration
+#
+# Please review and edit where needed
+# Several dependencies, like ConfigMaps and Secrets, must be created too.
+---
+apiVersion: pais.vmware.com/v1alpha1
+kind: PAISConfiguration
+metadata:
+  name: default
+  namespace: my-namespace-on-supervisor
+spec:
+  worker:
+    storageClassName: my-preferred-storage-class
+  clientTls:
+    # These are optional
+    # Each should be a ConfigMap with a single "ca.crt" field
+    caBundleRefs:
+    - name: ca-trust-bundle-for-my-dsm
+    - name: ca-trust-bundle-for-my-harbor
+    - name: ca-trust-bundle-for-my-data-sources
+    # etc...
+  database:
+    host: my-postgres-with-pgvector-database-host-name
+    username: my-postgres-username
+    dbname: my-postgres-logical-db-name
+    # This is a required reference to a Kubernetes Secret
+    passwordRef:
```

Comment:
All referenced resources (K8s Secrets, ConfigMaps below) should be in the same namespace as the PAISConfiguration, right?